### PR TITLE
Portfolio: Portfolio mobile version

### PR DESCRIPTION
**Pull request by faustM**
* I created the mobile website section where I will list my portfolio projects.
* I also create a separate pull request

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -63,8 +63,117 @@ <h3 class="skill-title">Skills</h3>
...
             <div class="eli18"><img src="images/icones/Ellipse18.svg" alt=" img ackround "></div>
             <div class="eli19"><img src="images/icones/Ellipse19.svg" alt="img backgound "></div>
+  
         </div>
     </section>
+    <div class="work">
+        <h3>WORK</h3>
+      </div>
+    <div class="recent-work">
+      <h2 class="recent">My recent <br> <br>Work</h2>
```
Comment (Mre55) [inline index.html]: Kindly, match the design provided for this text and make it in uppercase and decrease the `font-weight` to match the design.  The horizontal line should be in the middle of the text and image. Please adjust the spacing for it.


| Your Design | Figma Design |
|-------------|---------------|
| image | image |

File: index.html [hunk 2]
```diff
@@ -63,8 +63,117 @@ <h3 class="skill-title">Skills</h3>
...
+    </div>
+  <hr>
+  <section class="project" id="aboutproject">
+    <div class="first-project child">
```
Comment (Mre55) [inline index.html:77]: As the project requirement, the main container of the projects has to be `display: grid` and you can use `Flexbox` for the specific cards.

image

File: index.html [hunk 3]
```diff
@@ -63,8 +63,117 @@ <h3 class="skill-title">Skills</h3>
...
+          </li>
+          <li class="tools-item right-border ">
+            <a class="white" href="#">html</a>
+          </li>
+          <li class="lile right-border ">
+            <a class="white" href="#">bootstrap</a>
+          </li>
+          <li class="tools-item ">
+            <a class="white"  href="#">Ruby</a>
+          </li>
```
Comment (Mre55) [inline index.html:128]: Kindly, add proper background images as `background-image` in your CSS file for the images which are on the back of the text like this to match the design provided. Also, align the your `tool-list` to the left as the design and edit the text color for the button.

| Your Design | Figma Design |
|-------------|---------------|
| image | image |

Hint: you can give the `background-image` `z-index` of a big negative value to display text above the image.

**Review by Mre55 — CHANGES_REQUESTED** [review]
**Hi @faustM,**

You have done a great job so far! 💪 👏 
You still need to work on a few issues before moving on to the next project, but you're almost there!

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The commit messages are meaningful ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 

## Required Changes ♻️🛠️ 
_Check the comments under the review._




## Optional suggestions 🟡 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -63,8 +63,117 @@ <h3 class="skill-title">Skills</h3>
...
+  
         </div>
     </section>
+    <div class="work">
+        <h3>WORK</h3>
+      </div>
+    <div class="recent-work">
+      <h2 class="recent">MY RECENT <br> WORK</h2>
+    </div>
+  <hr>
```
Comment (jaflih) [inline index.html:75]: As asked by the previous reviewer, correct the layout of the `<hr>`
You can use `margin-bottom: 50px` to increase the marge between the line and the image

image

**Review by jaflih — CHANGES_REQUESTED** [review]
Hi @faustM 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.




 ## Highlights

The Pull Request is following the GitHub Flow ✅
The commit messages are meaningful ✅
The README is well documented ✅
Linters are working properly ✅

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 4 above)
Comment (faustM) [reply index.html:75]: thank you!!

**Review by xtrahuman — APPROVED** [review]
Hi @faustM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**